paper.bib

```diff
@@ -0,0 +1,140 @@
+}
+
+@article{Bezanson_Julia_A_fresh_2017,
+author = {Bezanson, Jeff and Edelman, Alan and Karpinski, Stefan and Shah, Viral B.},
+doi = {10.1137/141000671},
+journal = {SIAM Review},
+month = sep,
+number = {1},
+pages = {65--98},
+title = {{Julia: A fresh approach to numerical computing}},
+volume = {59},
+year = {2017}
+}
+
+@misc{churavy2022bridging,
+      title={Bridging HPC Communities through the Julia Programming Language}, 
+      author={Valentin Churavy and William F Godoy and Carsten Bauer and Hendrik Ranocha and Michael Schlottke-Lakemper and Ludovic Räss and Johannes Blaschke and Mosè Giordano and Erik Schnetter and Samuel Omlin and Jeffrey S. Vetter and Alan Edelman},
+      year={2022},
+      eprint={2211.02740},
+      archivePrefix={arXiv},
+      primaryClass={cs.DC}
+}
+
+@ARTICLE{2020NumPy-Array,
+  author  = {Harris, Charles R. and Millman, K. Jarrod and
+            van der Walt, Stéfan J and Gommers, Ralf and
+            Virtanen, Pauli and Cournapeau, David and
+            Wieser, Eric and Taylor, Julian and Berg, Sebastian and
+            Smith, Nathaniel J. and Kern, Robert and Picus, Matti and
+            Hoyer, Stephan and van Kerkwijk, Marten H. and
+            Brett, Matthew and Haldane, Allan and
+            Fernández del Río, Jaime and Wiebe, Mark and
+            Peterson, Pearu and Gérard-Marchant, Pierre and
+            Sheppard, Kevin and Reddy, Tyler and Weckesser, Warren and
+            Abbasi, Hameer and Gohlke, Christoph and
+            Oliphant, Travis E.},
+  title   = {Array programming with {NumPy}},
+  journal = {Nature},
+  year    = {2020},
+  volume  = {585},
+  pages   = {357–362},
+  doi     = {10.1038/s41586-020-2649-2}
+}
+
+@InProceedings{petsc-efficient,
+  author	= {Satish Balay and William~D. Gropp and Lois Curfman McInnes and Barry~F. Smith},
+  title		= {Efficient Management of Parallelism in Object Oriented Numerical Software
+		  Libraries},
+  booktitle	= {Modern Software Tools in Scientific Computing},
+  editor	= {E. Arge and A.~M. Bruaset and H.~P. Langtangen},
+  publisher	= {Birkh{\"{a}}user Press},
+  pages		= {163--202},
+  year		= {1997}
+}
+
+@Article{dealii2019design,
+  title   = {The {deal.II} finite element library: Design, features, and insights},
+  author  = {Daniel Arndt and Wolfgang Bangerth and Denis Davydov and
+             Timo Heister and Luca Heltai and Martin Kronbichler and
+             Matthias Maier and Jean-Paul Pelteret and Bruno Turcksin and
+             David Wells},
+  journal = {Computers \& Mathematics with Applications},
+  year    = {2021},
+  DOI     = {10.1016/j.camwa.2020.02.022},
+  pages   = {407-422},
+  volume  = {81},
+  issn    = {0898-1221},
+  url     = {https://arxiv.org/abs/1910.13247}
+}
+
+@software{Carlsson_Ferrite_jl,
+author = {Carlsson, Kristoffer and Ekre, Fredrik and {Ferrite.jl contributors}},
+license = {MIT},
+title = {{Ferrite.jl}},
+url = {https://github.com/Ferrite-FEM/Ferrite.jl}
+}
```

Comment:
No year (which makes sense) results in "n.d." in the paper. Is there any way to avoid this?